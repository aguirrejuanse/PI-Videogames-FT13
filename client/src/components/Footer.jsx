import React from 'react';
import '../assets/components/Footer.scss'

const Footer = () => (
    <footer className="footer">
        <a href="https://www.linkedin.com/in/juanseaguirre/" target="_blank" rel="noreferrer" >Creado por Juan Aguirre</a>
    </footer>
)

export default Footer;