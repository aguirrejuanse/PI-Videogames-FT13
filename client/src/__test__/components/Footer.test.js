import React from 'react';
import { mount } from 'enzyme';
//snapshot
import { create } from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('<Footer />', () => {
    const footer = mount(<Footer />);
    
    it('Render del componente Footer', () => {
        expect(footer.length).toEqual(1);
    });

    it('Footer contiene un <a>', () => {
        expect(footer.find('a')).toHaveLength(1); 
    })
});

describe('Footer Snapshot', () => {
    it('Comprobar UI', () => {
        const footer = create(<Footer />);
        expect(footer.toJSON()).toMatchSnapshot();
    })
})
