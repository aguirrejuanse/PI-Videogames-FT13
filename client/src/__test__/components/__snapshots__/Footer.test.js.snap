// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot Comprobar UI 1`] = `
<footer
  className="footer"
>
  <a
    href="https://www.linkedin.com/in/juanseaguirre/"
    rel="noreferrer"
    target="_blank"
  >
    Creado por Juan Aguirre
  </a>
</footer>
`;
